foe-ios/View Controllers/SightingDetailViewController.swift

```diff
@@ -75,3 +119,13 @@ class SightingDetailViewController: UIViewController {
     */
 
 }
+
+extension String {
```

Comment:
I think we should avoid extending String. I also wrote a string formatting method in [DetailsFormViewController](https://github.com/uwblueprint/foe-ios/blob/master/foe-ios/View%20Controllers/DetailsFormViewController.swift#L159-L163), perhaps we could pull these into a utility class to be used everywhere.